packages/workflow/package.json

```diff
@@ -28,38 +28,38 @@
     "@availity/mock-server": "^7.0.0",
     "@availity/workflow-logger": "^5.5.0",
     "@availity/workflow-upgrade": "^5.4.2",
-    "@babel/core": "^7.0.0",
+    "@babel/core": "^7.7.2",
     "@babel/plugin-proposal-optional-chaining": "^7.6.0",
-    "@babel/runtime": "^7.0.0",
-    "@testing-library/jest-dom": "^4.0.0",
+    "@babel/runtime": "^7.7.2",
+    "@testing-library/jest-dom": "^4.2.3",
     "babel-core": "^7.0.0-bridge.0",
     "babel-jest": "^24.0.0",
     "babel-loader": "^8.0.6",
     "babel-plugin-jsx-remove-data-test-id": "^2.0.0",
     "babel-plugin-root-import": "^6.4.1",
     "babel-preset-react-app": "^9.0.2",
     "case-sensitive-paths-webpack-plugin": "^2.1.2",
-    "chalk": "^2.3.2",
-    "copy-webpack-plugin": "^5.0.4",
-    "cross-spawn": "^6.0.5",
-    "css-loader": "2.1.1",
```

Comment:
You should check `css-loader` as well. This broke for me previously when trying to upgrade.